.github/ISSUE_TEMPLATE/REPOSITORY_REQUEST_TEMPLATE.yaml

```
@@ -0,0 +1,123 @@
+---
+name: '📦Repository Request'
```

This is for a GitHub repo only, right? So name the file `GITHUB_REPOSITORY_REQUEST_TEMPLATE` and say it everywhere consistently.

```suggestion
name: '📦 New GitHub Repository Request'
```